packages/altair-app/src/app/modules/altair/services/electron-app/electron-app.service.ts

```diff
@@ -101,6 +101,16 @@ export class ElectronAppService {
       debug.log('Electron app connected.');
 
       this.ipc.send('get-file-opened');
+
+      this.store.select((state: any) => state.settings['alert.disableUpdateNotification'])
+        .pipe(first())
+        .subscribe((disableUpdateNotification: boolean) => {
+          if (!disableUpdateNotification) {
+            this.ipc.on('update-available', () => {
+              this.notifyService.info('Found new updates!, go to Update to get the latest version', 'Update Found!')
```

Comment:
How do I _go to Update_?

Reply:
Perhaps clicking the notification should have the same effect. i.e. clicking the notification should trigger the update.

Reply:
Also just looked at the screenshots. I'm concerned we now have both "Update" and "Check for updates.." menu items. That can get confusing.

Reply:
Yep, it's confusing, clicking the notification to get the update seems to be a better idea, if u are ok with that i'll make the change in no time :D

Reply:
I've made the change, now there's no "update" option next to "check for updates" and when you click on the notification it downloads the update